lib/Settings/SettingsLabel/SettingsLabel.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { AppIcon } from '@folio/stripes/core';
+
+const SettingsLabel = ({
+  messageId,
+  iconKey,
+  app,
+}) => {
+  return (
+    <div data-test-setting-label>
```

Comment:
For consistency with component name.
```suggestion
    <div data-test-settings-label>
```